cmd/podman/images/pull.go

```diff
@@ -32,8 +32,13 @@ var (
 
 	// Command: podman pull
 	pullCmd = &cobra.Command{
-		Use:               "pull [options] IMAGE",
-		Args:              cobra.ExactArgs(1),
+		Use: "pull [options] IMAGE",
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
Can you use `Args: cobra.MinimumNArgs(1)` instead?  This will do the same but be consistent with the error messages of other commands.

Reply:
Oh right, thanks!